src/common/hotbar-store.ts

```diff
@@ -23,8 +23,8 @@ import { action, comparer, observable, toJS } from "mobx";
 import { BaseStore } from "./base-store";
 import migrations from "../migrations/hotbar-store";
 import * as uuid from "uuid";
-import { CatalogEntityItem } from "../renderer/components/+catalog/catalog-entity.store";
```

Comment:
Why changed: it's not good practice to import from renderer to common. This change also simplifies things (see tests).